src/api/api.cc

```diff
@@ -11935,29 +11935,40 @@ void recordreplay::SetRecordingOrReplaying(void* handle) {
 
   // Set flags to disable non-deterministic posting of tasks to other threads.
   // We don't support this yet when recording/replaying.
-  internal::FLAG_concurrent_array_buffer_sweeping = false;
-  internal::FLAG_concurrent_marking = false;
-  internal::FLAG_concurrent_sweeping = false;
-  internal::FLAG_incremental_marking_task = false;
-  internal::FLAG_parallel_compaction = false;
-  internal::FLAG_parallel_marking = false;
-  internal::FLAG_parallel_pointer_update = false;
-  internal::FLAG_parallel_scavenge = false;
-  internal::FLAG_scavenge_task = false;
+  if (V8RecordReplayFeatureEnabled("v8-gc-tasks", nullptr)) {
```

Comment:
V8 flags we are messing with
cc @kannanvijayan @klochek

Reply:
Note this PR isn't altering the flags we set by default.